ddsketch/store.py

```diff
@@ -30,7 +30,7 @@ def add(self, key):
         necessary."""
 
     @abstractmethod
-    def key_at_rank(self, rank):
+    def key_at_rank(self, rank, reverse=False):
```

Comment:
Why don't we do as in https://github.com/DataDog/sketches-py/pull/12#discussion_r517199872?

Reply:
I'm not sure what you mean... Specifying reverse does "#12 (comment)" no?

Reply:
I meant removing `reverse` as a method or a parameter from the interface, so that we don't have to handle it in implementations, as we can easily do it in `DDSketch`, once for all. But that's not a big deal.